release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -598,6 +598,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-node-bug-fixes"]
 ==== Node
 
+* Previously, the `LowNodeUtilization` strategy, which is enabled by the `LifecycleAndUtilization` descheduler profile, did not support namespace exclusion. With this release, namespaces are excluded properly when the `LifecycleAndUtilization` descheduler profile is set. (link:https://issues.redhat.com/browse/OCPBUGS-513[*OCPBUGS-513*])
```

Comment:
FYI @JoeAldinger. Currently I don't think this bug shows up in your RN queries. But just in case it does in the future, the entry is being added already here